### DISCUSSION TITLE
error message in chinese when we submitted

### DISCUSSION THREAD

**Discussion by begovegar** [PaddlePaddle/PaddleSpatial#176] [BaiduKDDCup2022] [answered]
Hi, we have made a submission resulting in "error ocurred" but we only see this message in chinese.
异常
Can you please let us know which indicates this error? 
We just changed the parameter "gpu": 1 in the settings dictionary in the prepare.py because your enviroment is prepared to work with a GPU, right? Unfortunately we have trained our model with a CPU, so we set this parameter to 0 in the training to generate the model.
Is it possible that the error is due to this?
 

![image](https://user-images.githubusercontent.com/104911705/173823276-2e16fd94-ad46-445f-9281-47058ddc9480.png)

**Comment by ramber1836** [comment, marked as answer]
Only one GPU is available, so the GPU id should be 0.
